common-content/en/module/decomposition/backend-statefulness/index.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Backend statefulness"
+headless = true
+time = 30
+facilitation = false
+emoji= "📖"
+objectives = [
+    "Identify whether some program is stateful or stateless.",
+]
++++
+
+Our example backend is **stateless**. If you make several requests to it, it will always do the same thing (even though doesn't always return exactly the same result!).
```

Comment:
tooltip on stateless

Reply:
The sentence that follows is the definition of stateless I'm using. I'm not sure hiding that sentence in a tooltip is more clear than just stating it in the flow of the text?

Reply:
I agree don't hide it: I meant double define it.